stdlib/LinearAlgebra/src/uniformscaling.jl

```diff
@@ -174,7 +174,7 @@ function (-)(J::UniformScaling, A::AbstractMatrix)
 end
 
 inv(J::UniformScaling) = UniformScaling(inv(J.λ))
-norm(J::UniformScaling, p::Real=2) = abs(J.λ)
+opnorm(J::UniformScaling, p::Real=2) = opnorm(J.λ, p)
```

Comment:
Why not `abs(J.λ)`?   This should be correct for all `p` since `J.λ` is a scalar, no?  (Do we even have an `opnorm` method for scalars?)

Reply:
I thought someone might want to define some unusual number system with a strange norm as `abs` that is suited to the specific use case but does not yield a Banach algebra. But this might be rather unusual, so I could replace `opnorm` with `abs`. There is method `opnorm(x::number, p::Real = 2)` in https://github.com/JuliaLang/julia/pull/27401/files#diff-e5541a621163d78812e05b4ec9c33ef4R559.

Reply:
It is hard for me to imagine such an algebra, and `norm(x::Number)` defaults to `abs(x)` anyway, but I guess it doesn't matter as long as the `opnorm` method is defined, since performance is not critical here.